tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/MemberList.kt

```diff
@@ -0,0 +1,90 @@
+package net.corda.cli.plugins.network
+
+import net.corda.cli.plugins.common.RestClientUtils.createRestClient
+import net.corda.cli.plugins.common.RestCommand
+import net.corda.membership.rest.v1.MemberLookupRestResource
+import net.corda.membership.rest.v1.types.response.RestMemberInfo
+import picocli.CommandLine
+
+@CommandLine.Command(name = "members", description = ["List members"])
```

Comment:
This changes the command to `network members members-list`. We should do
```suggestion
@CommandLine.Command(name = "members-list", description = ["Shows the list of members on the network."])
```